src/App.js

```diff
@@ -12,42 +12,100 @@ import React from "react";
 // components
 //=============================
 
-//=============================
-// containers
-//=============================
 
-//=============================
-// hooks
-//=============================
+export class AddTodo extends React.Component {
 
-//=============================
-// styles
-//=============================
+    updateInput = (e) => {
+        console.log(e);
+    };
+
+    render() {
+        return (
+            <div className='addTodoContainer'>
+                <form>
+                    <input
+                        onChange={(e) => this.updateInput(e)}
+                        type='text'>
+                    </input>
```

Comment:
The benefit of having this spread across lines is that, the multi-line spread makes it apparent if a `space` has crept between `<input> </input>` which makes the JSX invalid.

There should be NO SPACE between the `input` tags.